calabash.xcodeproj/project.pbxproj

```diff
@@ -122,7 +122,7 @@
 		B15BF9BB19ABB1DD00B38577 /* LPKeychainRoute.m in Sources */ = {isa = PBXBuildFile; fileRef = F55D412118C9130100813F78 /* LPKeychainRoute.m */; };
 		B15BF9BC19ABB1DD00B38577 /* LPDebugRoute.m in Sources */ = {isa = PBXBuildFile; fileRef = B12DCD84189A997B00F8DF87 /* LPDebugRoute.m */; };
 		B15BF9BD19ABB1DE00B38577 /* LPAppPropertyRoute.m in Sources */ = {isa = PBXBuildFile; fileRef = B1EBF08516EE63A000AE4D40 /* LPAppPropertyRoute.m */; };
-		B15BF9BE19ABB1DE00B38577 /* LPQueryLogRoute.m in Sources */ = {isa = PBXBuildFile; fileRef = B1EBF08716EE63A000AE4D40 /* LPQueryLogRoute.m */; settings = {COMPILER_FLAGS = "-fobjc-arc"; }; };
+		B15BF9BE19ABB1DE00B38577 /* LPQueryLogRoute.m in Sources */ = {isa = PBXBuildFile; fileRef = B1EBF08716EE63A000AE4D40 /* LPQueryLogRoute.m */; settings = {COMPILER_FLAGS = "-fobjc-arc -Wno-deprecated-declarations"; }; };
```

Comment:
`-Wno-deprecated-declarations` means "don't warn me about deprecated methods or declarations".  I enabled that setting for several files and CocoaAsyncSocket and CocoaHttpServer.  I want a project with zero warnings.  I made Calabash iOS issues for anything that I added `-Wno-deprecated-declarations`.

We don't have anything in the server that is deprecated per se.  The official deprecations are done on the client.


After bumping the minimum iOS deployment target from 4.2 to 6.0 (required by BITCODE), I found a bunch of deprecated calls and resolved them either by using the current replacement or ignoring the warning (and making a Calabash iOS issue).